mapbox/services/directions.py

```diff
@@ -8,6 +8,10 @@
 from mapbox.services.base import Service
 from mapbox import errors
 
+try:
+    basestring
+except NameError:
+    basestring = str
```

Comment:
Cool, let's think about adding a compat module next time we need to treat another 2/3 compatibility issue.

Reply:
I'll go ahead and so that now while we're thinking about it!